.github/CONTRIBUTING.md

```diff
@@ -44,3 +44,7 @@ A lot of good work has already been done in project mechanics (requirements.txt,
 1. Submit a github issue.   
 2. We'll agree on the feature scope.     
 3. Submit a PR! (with updated docs and tests 🙃).   
+
+## Coding Styleguide
+1. Test the code with flake8.
```

Comment:
Rather say that we follow PEP8 which can be simply tested locally by `pylint` or `flake8`

Reply:
i remember its a little bit different from autopep8, but i think its okay to change this line 👍